x-pack/plugins/security_solution/public/management/components/artifact_entry_card/utils/is_trusted_app.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AnyArtifact } from '../types';
+import { TrustedApp } from '../../../../../common/endpoint/types';
+
+/**
+ * Type guard for `AnyArtifact` to check if it is a trusted app entry
+ * @param item
+ */
+export const isTrustedApp = (item: AnyArtifact): item is TrustedApp => {
+  return 'effectScope' in item;
```

Comment:
This will work now, but when we migrate TA to the exceptions API this will disappear and we will have to use the `list_id` (not sure if list_id is returned by the TA API).

Reply:
yeah, I think once we move to the type being `ExceptionLIstItemSchema` we will not need to have type guards in place because there will be only 1 type. But I also just checked, and yes, the list id is returned, so could use that to differentiate if needed:

<img width="609" alt="Screen Shot 2021-09-30 at 9 24 26 AM" src="https://user-images.githubusercontent.com/56442535/135463649-5b9fc5de-6e31-4830-a178-2865678a997b.png">